File: 110algoritmos/exercicios009/ex009-Csharp/ex009.cs
using System;

class Program {
    static void Main() {
        int num;

        Console.Write("Digite um número para tabuada: ");
        num = int.Parse(Console.ReadLine());

        Console.WriteLine("=-=-=-=-=-=");
        for(int i = 1; i<=10; i++)
        {
            Console.WriteLine(num + " x " + i + " = " + (num * i));
        }
        Console.WriteLine("=-=-=-=-=-=");


    }
}

File: 110algoritmos/exercicios009/ex009-Rust/ex009.rs
use std::io;

fn main() {

    let mut num:String = String::from("");

    println!("Digite um valor para tabuada: ");
    io::stdin()
    .read_line(&mut num)
    .expect("Falha ao ler o numero");
    let num:i32 = num.trim().parse().expect("Erro na conversao de num");
    
    println!("=-=-=-=-=-=");
    for n in 1..11 {
        println!("{} x {} = {}", num, n, (num*n));
    }
    println!("=-=-=-=-=-=");
}